src/Cli/Executable.php

```diff
@@ -0,0 +1,407 @@
+<?php
+
+namespace AmpProject\Cli;
+
+use AmpProject\Exception\AmpCliException;
+use AmpProject\Exception\Cli\InvalidSapi;
+use Exception;
+
+/**
+ * This file is adapted from the splitbrain\php-cli library, which is authored by Andreas Gohr <andi@splitbrain.org> and
+ * licensed under the MIT license.
```

Comment:
Would be useful to link to the specific file and revision on GitHub where this came from to incorporate future changes.

Reply:
```suggestion
 * This file is adapted from the splitbrain\php-cli library, which is authored by Andreas Gohr <andi@splitbrain.org> and
 * licensed under the MIT license.
 *
 * Source: https://github.com/splitbrain/php-cli/blob/fb4f888866d090b10e3e68292d197ca274cea626/src/CLI.php
```